File: 백준/Silver/1269. 대칭 차집합/대칭 차집합.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int Asize = Integer.parseInt(st.nextToken());
        int Bsize = Integer.parseInt(st.nextToken());

        Set<Integer> Aset = new HashSet<>();
        Set<Integer> Bset = new HashSet<>();

        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < Asize; i++) {
            Aset.add(Integer.parseInt(st.nextToken()));
        }
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < Bsize; i++) {
            Bset.add(Integer.parseInt(st.nextToken()));
        }
        br.close();

        int count = 0;
        for (int A : Aset) {
            if (!Bset.contains(A))
                count++;
        }
        for (int B : Bset) {
            if (!Aset.contains(B))
                count++;
        }

        System.out.println(count);
    }
}

File: 백준/Silver/1269. 대칭 차집합/대칭 차집합.py
import sys

input = sys.stdin.readline

Asize, Bsize = map(int, input().split())

Aset = set(map(int, input().split()))
Bset = set(map(int, input().split()))
count = 0

for A in Aset:
    if not A in Bset:
        count += 1

for B in Bset:
    if not B in Aset:
        count += 1
        
print(count)